From 72440e7ddf707291c90f8cb78728b322201c54a2 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 8 Mar 2018 22:48:21 +0100
Subject: [PATCH 1/1] wandboard: remove superfluous include

No definition provided by input.h is used in the board file.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 board/wandboard/wandboard.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/board/wandboard/wandboard.c b/board/wandboard/wandboard.c
index 1e7c11e670..b7c6c6cb18 100644
--- a/board/wandboard/wandboard.c
+++ b/board/wandboard/wandboard.c
@@ -28,7 +28,6 @@
 #include <miiphy.h>
 #include <netdev.h>
 #include <phy.h>
-#include <input.h>
 #include <i2c.h>
 #include <power/pmic.h>
 #include <power/pfuze100_pmic.h>
-- 
2.11.0

